### DISCUSSION TITLE
Cannot generate client

### DISCUSSION THREAD

**Discussion by Melonendk** [prisma/prisma#9383] [Q&A] [answered]
Hello im trying to generate the prisma client, but i am unable to do so as im getting this error.
Using prisma 3.1.1 on windows with latest node and npm versions
```cmd
C:\path\project>npx prisma generate
Prisma schema loaded from prisma\schema.prisma
Error: Unable to require(`C:\path\project\node_modules\prisma\query_engine-windows.dll.node`)
The specified module was not found.
\\?\C:\path\project\node_modules\prisma\query_engine-windows.dll.node
```

Even tho i havent changed anything from last time i did it.

**Comment by ryands17** [comment, marked as answer]
@Melonendk 👋 

A similar issue was faced and the workaround is specified in the comment. Could you try that and check?
https://github.com/prisma/prisma/issues/9304

**Reply by Melonendk** [reply]
Thanks that was what i missed :)

All tho i found the thing needed to be installed manually if people dont want chokolaty  i have commented under :)

**Comment by Melonendk** [comment]
So windows solution as it was the platform i had the issue, it seems to be the requirements

So make sure you check the box on the node install else redownload node and change the installaion and click install requirements.

Instead if you want to install manually you only need:

- Build tools from Visual studio community where you can find it [here](https://visualstudio.microsoft.com/downloads/)
- As for compilers install the C++ redistributable packages 2015-2019 in x86 and x64 you can find a downlaod to them [here](https://support.microsoft.com/en-us/topic/the-latest-supported-visual-c-downloads-2647da03-1eea-4433-9aff-95f26a218cc0)
- I cant say you need it for sure but download the latest python from microsoft store and you should be good :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
